integration-cli/docker_api_images_test.go

```diff
@@ -121,3 +121,26 @@ func (s *DockerSuite) TestApiImagesDelete(c *check.C) {
 	c.Assert(status, check.Equals, http.StatusOK)
 	c.Assert(err, check.IsNil)
 }
+
+func (s *DockerSuite) TestApiImagesHistory(c *check.C) {
```

Comment:
you need to add `testRequires(c, Network)` in order for this test to work on --net none build